src/connector/src/parser/unified/debezium.rs

```diff
@@ -165,13 +165,29 @@ pub fn parse_schema_change(
 ) -> AccessResult<SchemaChangeEnvelope> {
     let mut schema_changes = vec![];
 
-    let upstream_ddl = accessor
+    let upstream_ddl: String = accessor
         .access(&[UPSTREAM_DDL], &DataType::Varchar)?
         .to_owned_datum()
         .unwrap()
         .as_utf8()
         .to_string();
 
+    // Currently only accept ADD COLUMN and DROP COLUMN,
+    // and we assumes each schema change message only contains one DDL statement.
+    let allowed_ddl = ["ADD COLUMN", "DROP COLUMN"];
+    let upper_upstream_ddl = upstream_ddl.to_uppercase();
+    let is_allowed = allowed_ddl
+        .iter()
+        .any(|&allowed_ddl| upper_upstream_ddl.contains(allowed_ddl));
```

Comment:
> This looks fragile.

+1.
What if there are both `ALTER` and `ADD/DROP COLUMN` statements in `upstream_ddl`? From debezium's doc: "The ddl field can contain multiple DDL statements."

Reply:
> The ddl field can contain multiple DDL statements.

I am aware of this statement, but I cannot construct this case even with transaction. And it seems mysql doesn't support DDL transactions. So actually the parser assumes each message only contains a single DDL.